wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -109,13 +109,15 @@
         },
         "rdshUseManagedDisks": {
             "type": "bool",
+            "allowedValues": [true],
             "defaultValue": true,
             "metadata": {
                 "description": "Warning: Parameter 'rdshUseManagedDisks' will not be allowed to be false, starting March 1st, 2020. True indicating you would like to use managed disks or false indicating you would like to use unmanaged disks."
             }
         },
         "storageAccountResourceGroupName": {
             "type": "string",
+            "allowedValues": [""],
```

Comment:
Can this also support null?

Reply:
It looks like calling the template with `null` value for a param from the cmd line or params file is an error during validation. So it looks like there is no way to call this with `null` & this makes sense because there are params with default value as `""` & not `null`